MatomoTracker.php

```diff
@@ -49,27 +47,21 @@ class MatomoTracker
      *
      * @ignore
      */
-    const LENGTH_VISITOR_ID = 16;
+    public const LENGTH_VISITOR_ID = 16;
 
     /**
      * Charset
      * @see setPageCharset
      * @ignore
      */
-    const DEFAULT_CHARSET_PARAMETER_VALUES = 'utf-8';
+    public const DEFAULT_CHARSET_PARAMETER_VALUES = 'utf-8';
 
     /**
      * See matomo.js
      */
-    const FIRST_PARTY_COOKIES_PREFIX = '_pk_';
-
-    /**
-     * Defines how many categories can be used max when calling addEcommerceItem().
-     * @var int
-     */
-    const MAX_NUM_ECOMMERCE_ITEM_CATEGORIES = 5;
```

Comment:
Is this not used or why are we removing the constant?

Reply:
@michalkleiner yes, not used) At least I didn't find it in the library.

Reply:
Other plugins may still rely on this so if we didn't want to do a BC breaking change we need to keep the constant.

Reply:
@michalkleiner ooo, didn't know about plugins that they can use this library. Just for my curious, do you have some examples or maybe documentation?

Maybe I didn't understand correctly, please, correct me. This constant is not used by the library at all. Maybe it should be checked in the `addEcommerceItem()` method, but not. In this PR (https://github.com/matomo-org/matomo-php-tracker/pull/22) it was added.

I can imagine only one case: some of the customers will extend MatomoTracker class and use this constant in his/her child class somehow.

Reply:
If we can keep the constant that would great. Then we can look at it separately outside of the this PR.

Reply:
@michalkleiner yes, it's not a problem)